jupyter-extension/src/plugins/mount/types.ts

```diff
@@ -86,6 +86,13 @@ export interface IMountPlugin {
   ready: Promise<void>;
 }
 
+export type PpsConfig = {
+  pipeline_name: string;
+  image: string;
+  requirements: string | null;
+  input_spec: any;
+};
+
 export type SameMetadata = {
```

Comment:
Can you fully eliminate the Same metadata and other related code?

Reply:
Yes, I can do it here, but I figured it would be nice to do this incrementally in another PR

Reply:
Cool, I'll add a future work section